### PR TITLE
Solar device list - show individual trackers and fix voltage and amp displays

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

**Comment by blammit** [comment]
Not ready for review, need to verify when MQTT fixes are in.

**Comment by blammit** [comment]
Seems to work okay, taking this PR out of draft mode.

**Comment by chriadam** [comment]
Unrelated to this PR, but when I drill down to a tracker History and click on the Charts view, if I switch between 7 day and 30 day chart there is a visual glitch where the height of the bars jumps high/out of geometry bounds, before settling to correct values.

File: pages/solar/SolarChargerPage.qml [hunk 1]
```diff
@@ -10,6 +10,7 @@ Page {
 	id: root
 
 	property var solarCharger
+	readonly property var singleTracker: solarCharger.trackers.count === 1 ? solarCharger.trackers.get(0).solarTracker : null
```
[the remark this answers is not in the document]
Comment (blammit) [reply pages/solar/SolarChargerPage.qml:13]: This is a `SolarCharger` type from `data/common`, so it's not possible unless we put all of those types into `components`. Do you think that's worthwhile? Could do as a separate task at some point.

**Comment by blammit** [comment]
Have noted the visual glitch issue in https://github.com/victronenergy/gui-v2/issues/591

File: pages/solar/SolarChargerPage.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (chriadam) [reply pages/solar/SolarChargerPage.qml:13]: Is data/common part of the Victron.VenusOS import?  If it is, why aren't these types known at compile time, given that we now have AOT compiled typedata?  @DanielMcInnes

**Review by chriadam — APPROVED** [review]

File: pages/solar/SolarChargerPage.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (DanielMcInnes) [reply pages/solar/SolarChargerPage.qml:13]: I think this is possible. I changed it to:
`readonly property SolarCharger singleTracker: solarCharger.trackers.count === 1 ? solarCharger.trackers.get(0).solarTracker : null
`
and it seems to work

[the remark this answers is not in the document]
Comment (blammit) [reply pages/solar/SolarChargerPage.qml:13]: Oooh! Nice, I didn't know the import changes meant that these types would automatically be available now.

[the remark this answers is not in the document]
Comment (blammit) [reply pages/solar/SolarChargerPage.qml]: Have updated it to be a QtObject type, as the type is not a SolarCharger in mock mode.

**Pull request merged**